components/vm/25-vms.tf

```diff
@@ -38,6 +38,8 @@ module "virtual-machines" {
 
   custom_script_extension_name = "HMCTSVMBootstrap"
   tags                         = module.ctags.common_tags
+  xdr_tags                     = "${local.xdr_tags.activityName},${local.xdr_tags.application},${local.xdr_tags.env}"
```

Comment:
still need to add the other tags from xdr_tags map. A nice solution to this i think is:

`xdr_tags = join(",", values(local.xdr_tags))`

Reply:
no other values it is just the local